packages/web3-validator/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "web3-validator",
-	"version": "1.0.2",
+	"version": "2.0.0-rc.0",
```

Comment:
Sorry, I could not figure out why the major version needed to be updated for web3-validator.

Reply:
https://github.com/web3/web3.js/pull/6331/files#diff-f426c14226d3a2e349465f5bccc8e595ee3b3cbc4edb6dfbd2f12f1968603554R125

changes are mentioned in validator change log. 



-   Types `ValidationError` and `JsonSchema` were changed (#6264)

-   Type `RawValidationError` was removed (#6264)